src/tests/conftest.py

```diff
@@ -13,6 +13,7 @@
 import vecs
 
 PYTEST_DB = "postgresql://postgres:password@localhost:5611/vecs_db"
+PYTEST_SCHEMA = "test_schema"
```

Comment:
the best way to test that escaping is done correctly in all places is to use a crazy schema name.

I tested basic operations with the schema name "esCape Me!" and the test suite fails.

To reproduce that, try:
```
foo: Collection = client.get_or_create_collection(name="foo", schema="esCape Me!", dimension=5)
```
and you'll get
```
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.InvalidSchemaName) schema ""esCape Me!"" does not exist
```